src/index.ts

```diff
@@ -15,3 +15,8 @@ export * from "./handlers/session-check-handler";
 // Models
 export * from "./models/http-solid-context";
 export * from "./session-manager";
+export * from "./in-memory-storage"
+// Mocks
+// TODO how to get rid of it and still have components generated? it breaks npm start
+export * from "./mocked-storage";
+export * from "./mocked-session-manager";
```

Comment:
We have thrown it here just to get it working. It breaks `npm start` with an error about importing `@jest/globals`.
Tomorrow we will look into building actual components and mocked components separately. I probably will require separate tsconfig and run of components generator.

Reply:
We have a separate branch, which solves this problem, by transitioning into a monorepo and having separate packages. https://github.com/elf-pavlik/sai-impl-service/tree/monorepo